crates/but-debugging/src/lib.rs

```diff
@@ -0,0 +1,109 @@
+use std::{path::Path, process::Command};
```

Comment:
Added some debugging utils that let you get a bit of a better idea of the git graph and trees.